models/train.js

```diff
@@ -0,0 +1,50 @@
+const pg = require('pg-promise')()
+
+const connection =  {
+  host: 'localhost',
+  port: 5432,
+  database: 'lg-metrorail-db',
+  user: 'jhallman5',
+}
+
+const db = pg(connection)
+
+const getTrainId = (station) => {
+  if (typeof station !== 'string') {
+    return 'Please insert current train station name.'
+  }
+  return db.one('SELECT id FROM trains WHERE current_station = $1', [station])
+    .then( response => {
+      return response
+    })
+    .catch( error => {
+      return error
+    })
+}
+
+const getTrainCapacityById = (id) => {
+  if (typeof id !== 'number') {
+    return "Please provide a train ID as an integer."
+  }
+  return db.one('SELECT capacity FROM trains WHERE id = $1', [id])
```

Comment:
doesn't look like your function is going to return as intended without doing the extra couple lines of a .then such as you did above.